common/src/Sync.php

```diff
@@ -6,55 +6,57 @@
 use Customer;
 use DateTime;
 use Doctrine\ORM\EntityManagerInterface;
-use Exception;
 use ModuleFrontController;
 use PrestaShop\Module\ProductComment\Entity\ProductComment;
 use PrestaShop\PrestaShop\Adapter\Validate;
 use Product;
-use Symfony\Component\HttpKernel\Exception\HttpException;
 
 class Sync extends ModuleFrontController {
 
     /** @var bool */
     public $ajax;
 
-    /**
-     * @throws Exception
-     */
     public function postProcess(): void {
         $request_data = trim(file_get_contents('php://input'));
         if (!$request_data) {
-            throw new HttpException(400, 'Empty request data');
+            http_response_code(400);
+            echo 'Empty request data.';
+            exit();
```

Comment:
Instead of repeating this 6 times, you can use a function 😄 

----

You can also use `die('your message');` instead of `echo` + `exit()`